ads/google/a4a/utils.js

```diff
@@ -65,19 +65,18 @@ export function isGoogleAdsA4AValidEnvironment(win, element) {
  * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4a
  * @param {string} baseUrl
  * @param {number} startTime
- * @param {number} slotNumber
  * @param {!Array<!./url-builder.QueryParameterDef>} queryParams
  * @param {!Array<!./url-builder.QueryParameterDef>} unboundedQueryParams
```

Comment:
Can we have some documentation on the difference between queryParams and unboundedQueryParams?